src/App.tsx

```diff
@@ -6,12 +6,13 @@ import {
   Link,
   BrowserRouter,
 } from "react-router-dom";
+import "./styles.css";
```

Comment:
css file? why?

Reply:
It is used to apply bg image for all the pages